hell/adventcalendar/2025/6/index.md

```diff
@@ -1,35 +1,151 @@
 ---
-title: "Accessible by Design: The Role of the lang Attribute"
-author: "Your Name"
-author_bio: "Your short bio"
+title: "Accessible by Design: The Role of the 'lang' Attribute"
+author: "Todd Libby"
+author_bio: "Current desert nomad, lobster aficionado, long-time developer, founder of RouteReady, and W3C Invited Expert."
 date: 2025-12-06
 author_links:
   - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
-intro: "<p>Short introductory text</p>"
+    url: "https://toddl.dev"
+    link_label: "unfiltered"
+  - label: "Bluesky"
+    url: "https://bsky.app/profile/toddl.dev"
+    link_label: "@toddl.dev"
+  - label: "Mastodon"
+    url: "@colabottles@notacult.social"
+    link_label: "@colabottles"
+  - label: "GitHub"
+    url: "https://github.com/colabottles"
+    link_label: "Todd @ GitHub"
+intro: "<p>Sometimes, a tiny detail can make or break the experience for millions of users. One of these tiny, powerful details is the language attribute in your HTML.</p>"
 image: "advent25_6"
 ---
-Some text.
-Some text.
 
-Some text. Some text.
+Let&apos;s face it, in web development, we focus on making things for ourselves and if _it works on our computer, it must work everywhere&excl; Right&quest;_ I used to see it rarely when a developer or framework would have the `lang` attribute used in their work.
```

Comment:
I'm not saying that you must change anything here, but the switch from this general statement to usage of the lang attribute felt very abrupt.